digits/templates/models/images/classification/show.html

```diff
@@ -139,9 +137,36 @@ <h4 class='text-center'>Dataset</h4>
 updateSnapshotList({% autoescape false %}{{task.snapshot_list()}}{% endautoescape %});
                             </script>
                         </div>
+                        {% if show_multi_gpu_form %}
+                        <button
+                            formaction="{{url_for('digits.model.views.download', job_id=job.id())}}"
+                            formmethod="post"
+                            formenctype="multipart/form-data"
+                            class="btn btn-info">
+                            Download Model
+                        </button>
+                        <button
+                            formaction="{{url_for('digits.model.views.to_pretrained', job_id=job.id())}}"
+                            formmethod="post"
+                            formenctype="multipart/form-data"
+                            class="btn btn-success">
+                            Make Pretrained Model
+                        </button>
+                        {% endif %}
+                    </div>
+                    {% if show_multi_gpu_form %}
+                    <div class="col-sm-6">
+                        <div class="form-group{{mark_errors([form.select_one_of_gpus])}}">
+                            {{form.select_one_of_gpus.label}}
```

Comment:
I've [changed](https://github.com/NVIDIA/DIGITS/pull/1511/commits/019414fd0ae849b7970dbba2c4f08966ed201efe#diff-41696ab7b1c04e66a23ac76fd94bb75f) that too. What you think?